### PR TITLE
landing page

### PR DESCRIPTION
**Pull request by jwang711**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (thesteau) [inline .idea/.gitignore:1]: We need to add the following to the .gitignore folders, else the project would include these extraneous folders.

.idea/
.vscode/
.DS_Store

File: website/views.py [hunk 2]
```diff
@@ -53,3 +53,11 @@ def settings():
     Render Settings Page
     """
     return render_template("settings/settings.html")
+
+
+@views.route('/home')
```
[the remark this answers is not in the document]
Comment (jwang711) [reply website/views.py:58]: @jeffacuario would it be okay with you if I make these changes?

[the remark this answers is not in the document]
Comment (jeffacuario) [reply website/views.py:58]: Yeah that’s my mistake for taking “/“. I can make the adjustments for my job route after

[the remark this answers is not in the document]
Comment (jwang711) [reply website/views.py]: > Yeah that’s my mistake for taking “/“. I can make the adjustments for my job route after

Thanks Jeffrey!

File: .idea/.gitignore (hunk 1 above)
Comment (jwang711) [reply .idea/.gitignore]: This is done, please let me know if this is good to you.

File: website/views.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (thesteau) [reply website/views.py]: All good

File: .idea/.gitignore (hunk 1 above)
Comment (thesteau) [reply .idea/.gitignore]: All good

**Pull request merged**